Source/WebCore/page/TextIndicator.h

```diff
@@ -120,6 +120,9 @@ enum class TextIndicatorOption : uint16_t {
     // By default, TextIndicator does not consider the user-select property.
     // If this option is set, expand the range to include the highest `user-select: all` ancestor.
     UseUserSelectAllCommonAncestor = 1 << 12,
+
+    // If this option is set, exclude all content that is replaced by a seperate render pass, like images, media, etc.
```

Comment:
Nit - typo here too:

```suggestion
    // If this option is set, exclude all content that is replaced by a separate render pass, like images, media, etc.
```